packages/core/core.ts

```diff
@@ -165,18 +164,43 @@ function logInfoAboutFuzzerOptions(fuzzerOptions: string[]) {
 	});
 }
 
+function makeSigintCallback(options: Options, sync: boolean) {
```

Comment:
As this function is only used in `startFuzzingNoInit` it could be changed to an inner function to be more local to its usage. I wonder if that would be cleaner.

Reply:
With a comment explaing the locality, I agree!